.travis.yml

```diff
@@ -59,7 +59,7 @@ env:
     - PGVERSION=9.0 PQTEST_BINARY_PARAMETERS=no
 
 script:
- - go test -v ./...
+ - result=$(gofmt -d -e $(find -name \*.go)); test -z "$result" || (echo "$result" && false) && go test -v ./...
```

Comment:
@cbandy I'm going to merge as is since this seems to be working, but feel free to submit a PR with the awk mechanism. If you can get a PR to fail (by intentionally introducing an error) and then pass, I don't particularly care what the command looks like, and awk seems simpler here.